fastlane/Fastfile

```diff
@@ -659,5 +652,5 @@ lane :check_pods do
 end
 
 def current_version_number
-  File.read("../.version").strip
+  File.read("#{File.dirname(__FILE__)}/../.version").strip
```

Comment:
This was slightly wrong, now it works when called from any folder.

Reply:
yeah, my bad, I didn't really consider paths other than calling from root. Solid improvement

Reply:
It wasn't used from any of the testers before :)